modules/learn/pages/clusters-and-availability/groups.adoc

```diff
@@ -68,13 +71,16 @@ In each illustration, all servers are assumed to be running the Data Service.
 [#vbucket-distribution-across-equal-groups]
 === Equal Groups
 
-The following illustration shows how vBuckets are distributed across two groups; each group containing four of its cluster's eight nodes.
+The following illustration shows how vBuckets are distributed across three groups; two group containing four of its cluster's eight nodes and a thrid group that can include a single arbiter node..
```

Comment:
The original description was correct since the vBuckets are distributed only across two groups.  The third group only contains an arbiter node to allow a quorum to be formed if all the nodes in one server group fails.   So, should say:

The following illustration shows how vBuckets are distributed across two groups; each group containing four of the cluster's nodes.  The third group only contains one node, an arbiter node, which exists to allow a quorum to be formed if all the nodes in server group 1 or 2 fails.